matrix.py

```diff
@@ -1,29 +1,32 @@
-import math
-import random
-from msvcrt import getch
-
+import math #dla operacji matematycznych
+import random	# dla generowania losowych plansz
+from msvcrt import getch	#dla interface z użytkownikiem (nieużywane)

```

Comment:
Jeśli nieużywane to w sumie można skasować :)

Reply:
A w sumie prawda :)